test/realtime/resume.test.js

```diff
@@ -629,5 +624,87 @@ define(['shared_helper', 'async', 'chai'], function (helper, async, chai) {
         closeAndFinish(done, [sender_realtime, receiver_realtime, resumed_receiver_realtime], err);
       }
     });
+
+    // Tests recovering multiple channels only receives the expected messages.
+    it('recover multiple channels', async function () {
```

Comment:
We don't use async/await in tests and if we need to use the Promise API we wrap it in an `if (typeof Promise !== 'undefined') just so that the tests run in older browsers. I know it's a pain writing tests using the `async` library so i'd be fine with just using `Promise.then` but as it stands i think this will break all of the tests on IE because of unsupported syntax

Reply:
Just to be clarify do you mean wrap in `if (typeof Promise !== 'undefined')` and replace with `Promise.then` - or remove promises entirely and use `async` library?

Reply:
either would be fine, but the former would probably be easier

Reply:
ok sure replaced with `async` library (`.then` was being a pain)